ecosystem/typescript/sdk_v2/tests/unit/account_address.test.ts

```diff
@@ -350,4 +349,10 @@ describe("AccountAddress other parsing", () => {
     expect(invalidReason).toBeUndefined();
     expect(invalidReasonMessage).toBeUndefined();
   });
+
+  it("compares equality with equals as expected", async () => {
```

Comment:
```suggestion
  it("compares equality with equals as expected", () => {
```

think you can get rid of the asyncs on some of these, right?